win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -837,24 +679,17 @@ static DWORD WINAPI chaos_thread_func(LPVOID lpThreadParameter)
 
     while (InterlockedAdd(&chaos_test_data->chaos_test_done, 0) == 0)
     {
-        int which_action = rand() * 4 / (RAND_MAX + 1);
+        int which_action = rand() * 2 / (RAND_MAX + 1);
         switch (which_action)
         {
             case 0:
-                // perform an open
-                (void)threadpool_open(chaos_test_data->threadpool);
-                break;
-            case 1:
-                // perform a close
-                (void)threadpool_close(chaos_test_data->threadpool);
-                break;
-            case 2:
                 // perform a schedule item
                 if (threadpool_schedule_work(chaos_test_data->threadpool, work_function, (void*)&chaos_test_data->executed_work_functions) == 0)
                 {
                     (void)InterlockedIncrement64(&chaos_test_data->expected_call_count);
                 }
-            case 3:
+                break;
+            case 1:
```

Comment:
>1 [](http://example.com/codeflow?start=17&length=1)

I think this would be nicer if modeled by an ENUM (like we have below for TEST_ACTION_CLEANUP_TIMER, etc.

This is a consider being that the code was already like that.

Reply:
After the `threadpool_schedule_work` would be deprecated, this test will go away or will have only `threadpool_schedule_work_item` in which case the switch case block won't be needed.